### PR TITLE
Added known issue to MJPEG cameras, moved customize polling interval to its own category, minor fixes

### PR DESCRIPTION
**Pull request by lindsaymarkward**
Added known issue to MJPEG cameras, moved customize polling interval to its own category, minor fixes

### CODE REVIEW

File: source/_cookbook/customize_polling_interval.markdown [hunk 1]
```diff
@@ -7,8 +7,8 @@ sidebar: true
 comments: false
 sharing: true
 footer: true
-ha_category: Example configuration.yaml
----
+ha_category: Customize Defaults
+-------------------------------
```
Comment (balloob) [inline source/_cookbook/customize_polling_interval.markdown]: Why did you add these extra dashes? Are you sure that jekyll picks that up correctly?

Comment (lindsaymarkward) [inline source/_cookbook/customize_polling_interval.markdown]: Those were automatically added by PHPStorm and the MultiMarkdown plugin... I didn't notice it wasn't supposed to be like that. I'll fix it.

**Comment by balloob** [comment]
Looks good :dolphin: You can merge it when comment addressed.
